v1/content.yaml

```diff
@@ -612,6 +612,80 @@ paths:
                 x-restbase-parentrevision: '{{x-restbase-parentrevision}}'
       x-monitor: false
 
+  /lint/{title}{/revision}:
```

Comment:
`lint` doesn't seem too informative to me. Perhaps `lintstatus` or `linterrors` ? Any suggestions @subbuss / @Pchelolo ?

Reply:
`lint` was chosen for consistency with the transform endpoint `transform/wikitext/to/lint`...

Reply:
Oh, excellent point @Pchelolo